spec/omniauth/strategies/untappd_spec.rb

```diff
@@ -62,7 +62,7 @@
     subject { strategy.raw_info }
     let(:access_token) { double('AccessToken', options: {}) }
     let(:response) { double('Response', parsed: parsed_response) }
-    let(:user_info_url) { 'http://api.untappd.com/v4/user/info' }
+    let(:user_info_url) { 'https://api.untappd.com/v4/user/info' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.